packer/install-dependencies.sh

```diff
@@ -198,6 +198,13 @@ for AZURE_CNI_NETWORKMONITOR_VERSION in ${AZURE_CNI_NETWORKMONITOR_VERSIONS}; do
     echo "  - ${CONTAINER_IMAGE}" >> ${RELEASE_NOTES_FILEPATH}
 done
 
+AZURE_NPM_VERSIONS="1.0.18-rc3"
```

Comment:
are we still waiting for a final release image?

Reply:
updated to v1.0.18!